app/src/org/commcare/activities/components/FormNavigationUI.java

```diff
@@ -244,6 +246,10 @@ private static void updateFloatingLabels(CommCareActivity activity,
             for (FloatingLabel type : labelTypes) {
                 if (type.getAppearance().equals(hint)) {
                     String widgetText = widget.getPrompt().getQuestionText();
+                    String markdownWidgetText = widget.getPrompt().getQuestionText();
+                    if(markdownWidgetText != null){
+                        widgetText = markdownWidgetText;
+                    }
```

Comment:
Not following why these additions are needed. 
`widgetText` and `markedownWidgetText` are initially set to the same thing:
`widgetText = widget.getPrompt().getQuestionText();`
`markdownWidgetText = widget.getPrompt().getQuestionText();`